docs/en/usage.rst

```diff
@@ -0,0 +1,564 @@
+Usage
+=====
+
+WMS
+---
+
+Find out what a WMS has to offer. Service metadata:
+
+.. code-block:: python
+
+  >>> from owslib.wms import WebMapService
```

Comment:
same as for `$`, without the `>>>` is easier to copy-paste by users

Reply:
skipping `usage.rst` ... it will be replaced with future PRs.